src/lib/components/particles/square-cut-corner-icon/index.jsx

```diff
@@ -0,0 +1,29 @@
+import defaultStyles from "./style.module.scss"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+export const SquareCutCornerIcon = ({cornerColor, squareColor, squareSize, cornerPartyAbbreviation, squarePartyAbbreviation, styles}) => {
+
+  styles = mergeStyles({...defaultStyles}, styles)
+
+  // expected size is 24px, with a corner size of 15px and a margin of 2px
+  let cornerSide = squareSize / 24 * 15;
```

Comment:
Will this break if `squareSize` is anything other than 24px? If so I think it we need to make it dynamic or remove the size prop.

Reply:
oh no - sorry the comment is maybe confusing. This is just to get the proportion of the corner right. It's about 15/24ths of the size of the total thing. Maybe I should rewrite as `let cornerSize = squareSize * 0.625` and delete the comment. 

changing the size in the story should show that it scales proportionately